Scripts/Level.cs

```diff
@@ -1,48 +1,69 @@
-using System;
-using Godot;
-
-public class Level : Spatial
-{
-    private static readonly float initialTime = 30.0f;
-    private GUI gui;
-
-    public float TimeLeft
-    {
-        get
-        {
-            return timeLeft;
-        }
-        set
-        {
-            timeLeft = value;
-
-            if (timeLeft <= 0)
-                GameOver();
-
-            if (gui != null)
-                gui.TimeLeftText.Text = FormatTime(timeLeft);
-        }
-    }
-    private float timeLeft;
-
-    public override void _Ready()
-    {
-        TimeLeft = initialTime;
-        gui = GetNode<GUI>("GUI");
-    }
-
-    public override void _Process(float delta)
-    {
-        TimeLeft -= delta;
-    }
-
-    private string FormatTime(float timeLeft)
-    {
-        return $"{Math.Floor(timeLeft)} s";
-    }
-
-    public void GameOver()
-    {
-
-    }
-}
+using System;
+using Godot;
+
+public class Level : Spatial
+{
+    private static readonly float initialTime = 30.0f;
+    private GUI gui;
+    private Spatial enemyContainer;
+    private readonly PackedScene bossScene = ResourceLoader.Load<PackedScene>("res://Scenes/Enemy/BigMeleeEnemy.tscn");
+
+    public float TimeLeft
+    {
+        get
+        {
+            return timeLeft;
+        }
+        set
+        {
+            timeLeft = value;
+
+            if (timeLeft <= 0)
+                GameOver();
+
+            if (gui != null)
+                gui.TimeLeftText.Text = FormatTime(timeLeft);
+        }
+    }
+    private float timeLeft;
+
+    public override void _Ready()
+    {
+        TimeLeft = initialTime;
+        gui = GetNode<GUI>("GUI");
+    }
+
+    public override void _Process(float delta)
+    {
+        TimeLeft -= delta;
+    }
+
+    private string FormatTime(float timeLeft)
+    {
+        return $"{Math.Floor(timeLeft)} s";
+    }
+
+    public void GameOver()
+    {
+
+    }
+
+    public void Win(HealthEntity entity)
```

Comment:
Is this `HealthEntity entity` required? Can remove unless Godot gives error if its absent.

Reply:
Yea it gives error